components/security-mgt/org.wso2.carbon.security.mgt/src/main/java/org/wso2/carbon/security/SecurityConstants.java

```diff
@@ -119,4 +119,66 @@
 
     private SecurityConstants(){}
 
+    /**
+     * Contains the Keystore management service related constants.
+     */
+    public static class KeyStoreMgtConstants {
+
+        public static final String FILTER_FIELD_ALIAS = "alias";
+        public static final String FILTER_OPERATION_EQUALS = "eq";
+        public static final String FILTER_OPERATION_STARTS_WITH = "sw";
+        public static final String FILTER_OPERATION_ENDS_WITH = "ew";
+        public static final String FILTER_OPERATION_CONTAINS = "co";
+
+        public static final String SERVER_TRUSTSTORE_FILE = "Security.TrustStore.Location";
+
+        /**
+         * Enum for Keystore management service related errors.
+         */
+        public enum ErrorMessage {
+
+            ERROR_CODE_RETRIEVE_KEYSTORE("KSS-50001",
```

Comment:
shall we define error codes according to the new convention? 
Ref "[Architecture] WSO2 Identity Server REST API Error Response Standardization"

Reply:
fixed